src/color/test/__snapshots__/readableColor.test.js.snap

```diff
@@ -0,0 +1,31 @@
+exports[`readableColor should return black given papayawhip 1`] = `"#000"`;
+
+exports[`readableColor should return black given red, #FF0000 1`] = `"#000"`;
+
+exports[`readableColor should return black given red, hsl(0, 100%, 50%) 1`] = `"#000"`;
+
+exports[`readableColor should return black given rgb(120,120,120) 1`] = `"#000"`;
+
+exports[`readableColor should return black given white 1`] = `"#000"`;
+
+exports[`readableColor should return black given white 2`] = `"#000"`;
+
+exports[`readableColor should return black given white hex, #fff 1`] = `"#000"`;
+
+exports[`readableColor should return black given white, rgb(255,255,255) 1`] = `"#000"`;
+
+exports[`readableColor should return white given black, #000 1`] = `"#fff"`;
+
+exports[`readableColor should return white given black, rgb(0,0,0) 1`] = `"#fff"`;
+
+exports[`readableColor should return white given black, rgba(0,0,0,0.1) 1`] = `"#fff"`;
+
+exports[`readableColor should return white given black, rgba(0,0,0,0.7) 1`] = `"#fff"`;
```

Comment:
For now we can leave this as is. As soon as #111 is done, we can validate it and choose to either throw a warning or an error when passed rgba/hla.